app/views/categories/_top_links.html.erb

```diff
@@ -2,7 +2,7 @@
 <ul class="category-link-list">
   <% category.links.each do |link| %>
     <li class="category-link-list__item">
-      <a class="category-link-list__item__link" href="<%= link['url'] %>"><%= link['text'] %></a>
+      <a class="category-link-list__item__link" href="<%= link['url'] %>"><%= link['text'] %><%= ',' unless link === category.links.last %></a>
```

Comment:
2 things. should the comma have a space after it like the css? and is `==` not sufficient as the equality check?